packages/ffe-account-selector-react/src/components/account-selector-multi/AccountSelectorMulti.md

```diff
@@ -6,49 +6,53 @@ initialState = {
     value: undefined,
 };
 
-<AccountSelectorMulti
-    accounts={[
-        {
-            accountNumber: '123456789101',
-            name: 'Brukskonto',
-            currencyCode: 'NOK',
-            balance: 1337,
-        },
-        {
-            accountNumber: '123456789102',
-            name: 'Sparekonto',
-            currencyCode: 'NOK',
-            balance: 13337,
-        },
-    ]}
-    id="account-selector-multi-1337"
-    locale="nb"
-    onAccountSelected={acc => {
-        const selectedAccounts = Array.isArray(state.selectedAccounts) 
-            ? state.selectedAccounts 
-            : [];
 
-        const filteredAccounts = selectedAccounts
-            .filter(a => a.accountNumber !== acc.accountNumber);
-        
-        const accountAlreadySelectedAndShouldBeRemoved =
-            filteredAccounts.length !== selectedAccounts.length;
+<React.Fragment>
+    <Label htmlFor="account-selector-multi">Velg konto</Label>
+    <AccountSelectorMulti
+        accounts={[
+            {
+                accountNumber: '123456789101',
+                name: 'Brukskonto',
+                currencyCode: 'NOK',
+                balance: 1337,
+            },
+            {
+                accountNumber: '123456789102',
+                name: 'Sparekonto',
+                currencyCode: 'NOK',
+                balance: 13337,
+            },
+        ]}
+        id="account-selector-multi"
+        locale="nb"
+        onAccountSelected={acc => {
+            const selectedAccounts = Array.isArray(state.selectedAccounts)
+                ? state.selectedAccounts
+                : [];
 
-        if (accountAlreadySelectedAndShouldBeRemoved) {
-            setState({ 
-                selectedAccounts: filteredAccounts,
-            });
-        } else {
-            setState({
-                selectedAccounts: [
-                    ...selectedAccounts,
-                    acc,
-                ],
-            });
-        }
-    }}
-    onBlur={f => f}
-    selectedAccounts={state.selectedAccounts}
-    value={state.value}
-/>;
+            const filteredAccounts = selectedAccounts
+                .filter(a => a.accountNumber !== acc.accountNumber);
+
+            const accountAlreadySelectedAndShouldBeRemoved =
+                filteredAccounts.length !== selectedAccounts.length;
+
+            if (accountAlreadySelectedAndShouldBeRemoved) {
+                setState({
+                    selectedAccounts: filteredAccounts,
+                });
+            } else {
+                setState({
+                    selectedAccounts: [
+                        ...selectedAccounts,
+                        acc,
+                    ],
+                });
+            }
+        }}
```

Comment:
Could it be an idea to pull these long functions and data structures out of the actual JSX and place them above?  This goes for all examples.